prime-router/src/main/kotlin/cli/ConvertValuesetsYamlToCSV.kt

```diff
@@ -0,0 +1,131 @@
+package gov.cdc.prime.router.cli
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.SerializationFeature
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import com.fasterxml.jackson.module.kotlin.KotlinFeature
+import com.fasterxml.jackson.module.kotlin.KotlinModule
+import com.fasterxml.jackson.module.kotlin.readValue
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.output.TermUi
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.types.file
+import gov.cdc.prime.router.ValueSet
+import java.io.File
+import java.io.FileOutputStream
+
+/**
+ * ConvertValuesetsYamlToCSV is the command line interface for the convert-valuesets-to-csv command.
+ *
+ * It takes the source YAML in sender-automation.valuesets and converts it to a parent/child table pair
+ * for use with the sender_automation_value_set and sender_automation_value_set_row lookup tables
+ *
+ * NOTE: this is intended as a temporary, developer-only tool to be used during the course of the Valuesets Editor UI project.
+ * Once the project is delivered to Production and found stable enough, this tool can/should be removed
+ *
+ * Usage:
+ *
+ *  ./prime convert-valuesets-to-csv -i ./metadata/valuesets/sender-automation.valuesets
+ *
+ */
+class ConvertValuesetsYamlToCSV : CliktCommand(
```

Comment:
Wondering if there is a need for this tool.  Could we just store/generate CSV files instead?  The lookup table CLI commands work on CSV as well.  If each value set has its own lookup table then you may not need this.

Reply:
@carlosfelix2 the sole purpose of this tool is _just_ to be able to respond to changes to the schema and data during development (like what you're suggesting right now 😂). Once the table structure is stable, this tool would go away entirely.